src/extension.ts

```diff
@@ -955,6 +955,44 @@ export function getUSSProfile(node: ZoweUSSNode) {
     return profile;
 }
 
+/**
+ * Rules of mapping:
+ *  1. Dont do this for the top level HLQ
+ *  2a. If a conventional file (no members) only test the LLQ
+ *  2b. If a conventional file (no members) default is to return label
+ */
+function appendSuffix(label: string): string {
+    const limit= 5;
+    const bracket = label.indexOf("(");
+    const split = (bracket > -1) ? label.substr(0, bracket).split(".", limit) : label.split(".", limit);
+
+    const doUntil = 0 ; // (bracket > -1) ? 0 : split.length - 2;
+    for (let i = split.length - 1 ; i > doUntil; i--) {
+        if (split[i] === "ASM" || split[i].indexOf("ASSEMBL") > -1 ) {
```

Comment:
I see you also made everything more deterministic.
Now `jasmine.applied.rexx.scripts` will show as `file-name.rexx`